### PR TITLE
Add attributes for creating new models to post.

### PR DESCRIPTION
**Pull request by gearedmonkey**
### Summary

I missed some things in the last iteration of this task. Needed to add the updated vars to creating a new model.

### Checklist

- [x] Did you run `npm test`?
- [x] Did you update/add documentation for new methods or changed functionality?

Finally, is it ready to be reviewed and merged: yes 🍏 

### How to Review

1. Checkout this branch
2. Run the server
3. Checkout consus-client branch=add-model-tffsomething
4. Start client
5. Click view all models
6. Verify all data shows as defined in the REST.md of this repo

### Links

Put links to Jira tickets, and anything else, here.

- [TFF-91](https://msoese.atlassian.net/browse/TFF-91)
- [Add models client](https://github.com/TheFourFifths/consus-client/pull/20)

### CODE REVIEW

File: docs/REST.md [hunk 1]
```diff
@@ -64,14 +64,31 @@ Create a model.
...
 
 ```json
 {
     "status": "success",
     "data": {
-        "address": "m8y7nEtAe"
+        "address": "m8y7nEtAe",
+        "name": "Resistor",
+        "description": "V = IR",
+        "manufacturer": "Pancakes R\' Us",
```
Comment (gearedmonkey) [inline docs/REST.md]: I didn't push my fix for this >.< need to wait till I get home tonight.

File: docs/REST.md [hunk 2]
```diff
@@ -64,14 +64,31 @@ Create a model.
...
 
 * `name`: The name of the model
+* `description`: The description of the model
+* `manufacturer`: The manufacturer of the model
+* `vendor`: The vendor who sold the model
+* `location`: Location where the model is stored
+* `isFaulty`: Whether the model is faulty of not
+* `faultDescription`: Description of the fault
+* `price`: Price of one model
+* `count`: Amount of this model in stock
```
Comment (thofmann) [inline docs/REST.md]: Attributes look good to me. :+1:

File: docs/REST.md (hunk 1 above)
Comment (thofmann) [inline docs/REST.md]: Please fix.

File: docs/REST.md [hunk 3]
```diff
@@ -64,14 +64,31 @@ Create a model.
...
     "status": "success",
     "data": {
-        "address": "m8y7nEtAe"
+        "address": "m8y7nEtAe",
+        "name": "Resistor",
+        "description": "V = IR",
+        "manufacturer": "Pancakes R\' Us",
+        "vendor": "Mouzer",
+        "location": "Shelf 14",
+        "isFaulty": "false",
```
Comment (thofmann) [inline docs/REST.md]: Should this be a string `"false"` or boolean `false`? JSON supports booleans `true` and `false`.

File: docs/REST.md [hunk 4]
```diff
@@ -92,7 +109,15 @@ Retrieve a model.
     "data": {
         "model": {
             "address": "m8y7nEtAe",
-            "name": "Transistor"
+            "name": "Resistor",
+            "description": "V = IR",
```
Comment (thofmann) [inline docs/REST.md]: Ω

File: docs/REST.md [hunk 5]
```diff
@@ -64,14 +64,31 @@ Create a model.
...
+        "address": "m8y7nEtAe",
+        "name": "Resistor",
+        "description": "V = IR",
+        "manufacturer": "Pancakes R\' Us",
+        "vendor": "Mouzer",
+        "location": "Shelf 14",
+        "isFaulty": "false",
+        "faultDescription": "",
+        "price": "10.50",
+        "count": "20"
```
Comment (thofmann) [inline docs/REST.md]: Should this be a string `"20"` or just the number `20`? JSON supports numbers.

File: docs/REST.md [hunk 6]
```diff
@@ -64,14 +64,31 @@ Create a model.
...
-        "address": "m8y7nEtAe"
+        "address": "m8y7nEtAe",
+        "name": "Resistor",
+        "description": "V = IR",
+        "manufacturer": "Pancakes R\' Us",
+        "vendor": "Mouzer",
+        "location": "Shelf 14",
+        "isFaulty": "false",
+        "faultDescription": "",
+        "price": "10.50",
```
Comment (thofmann) [inline docs/REST.md]: Should this be a string `"10.50"` or just the number `10.5`? JSON supports numbers.

**Pull request merged**